docs/ml/upload-model.md

```diff
@@ -43,7 +43,7 @@ Select **Next steps** to continue.
 1. Specify a name for the model.
 2. Specify the **Model type**.
 3. Add a `.tflite` model file.
-4. Add a `.txt` label file.
+4. (Optional) Add a `.txt` label file.
```

Comment:
just talking about this, if we can do some research on what label files should look like or link out to something that woudl be good

Reply:
Sure, here's a [common audio ML model](https://www.kaggle.com/models/google/yamnet/frameworks/tfLite/variations/classification-tflite/versions/1?tfhub-redirect=true), the download zip includes a good example labels.txt file. Looks like this:

```
bananajr:Downloads andf$ head -5 labels.txt
Person
Bicycle
Car
Motorcycle
Airplane
```

Reply:
This will usually be generated for them I think.